conda-recipe/meta.yaml

```diff
@@ -1,27 +1,30 @@
 {% set data = load_setup_py_data() %}
 
 package:
-    name    : nabs
-    version : {{ data.get('version') }}
+  name: nabs
+  version : {{ data.get('version') }}
 
 source:
-    path: ..
+  path: ..
+
+build:
+  number: 1
+  noarch: python
 
 requirements:
     build:
-      - python
+      - python >=3.6
+      - setuptools
 
     run:
-      - bluesky 
-      - python
+      - python >=3.6
+      - bluesky
```

Comment:
Looks like pandas, lmfit, scipy may be missing

Handy alias: `git grep -h 'import ' |sort |uniq`

Reply:
I don't see pandas or lmfit anywhere in the repo, but I do see the missing scipy dependency

Reply:
I was on the wrong branch locally - my apologies (`pswalker_upstream` from #22)

Reply:
Ah! That makes sense